app/controllers/api/cookies_consent_controller.rb

```diff
@@ -0,0 +1,23 @@
+# require 'ofn_web'
+
+module Api
+  class CookiesConsentController < BaseController
+    respond_to :json
+    include CookiesConsentHelper
+    # include Web::CookiesConsentHelper
+
+    def show
+      render json: { cookie_consent: CookiesConsent.new(cookies, request.host).exists? }
+    end
+
+    def create
+      CookiesConsent.new(cookies, request.host).set
+      show
```

Comment:
I'm not sure the small DRY is worth here. We are creating a new instance for the call to `render`. Won't this be troublesome if the class grows in the future and this goes unnoticed?

Reply:
sorry, I dont think I understand your point @sauloperez

Reply:
Sorry, I meant the DRY we get by calling `show` instead of directly calling `render`. It does create a new instance of `CookiesConsent`.

Reply:
oh, I didnt see it! now I see it.
I fixed it now.